learning/courses/quantum-chem-with-vqe/_toc.json

```
@@ -1,6 +1,6 @@
 {
   "parentUrl": "/learning",
-  "title": "Quantum Business Foundations",
+  "title": "Quantum Chemistry with VQE",
```

Abby C wants sentence case for course names

```suggestion
  "title": "Quantum chemistry with VQE",
```